### DISCUSSION TITLE
Can I send audio from one device and receive on may?

### DISCUSSION THREAD

**Discussion by pinkfloyd11** [pschatzmann/arduino-audio-tools#1485] [Q&A] [answered]
Hi friends, my case is:

I should transmit audio mono from one source to many receiver (broadcast).

This audio is trasmitted by ESP32 + ADC with jack input  connected  to a cd player.

1 - It is possible to do that whit this library? 

2 - Every ESP32 receiver must have display and keyboard for bluetooth association?

3 - If association must be done, it is keep into devices so it is done only one time?

Thanks very much and congratulation to developer for this monumental effort

Giuseppe

**Comment by pschatzmann** [comment, marked as answer]
Sure, but I don't think that Bluetooth would be part of the solution.
Have a look at the communication examples.  
I suggest that you look into UDP broadcast! 

You might need to use a codec to reduce the amount of data to be transmitted.
